docs/source/changelog.rst

```diff
@@ -1,11 +1,44 @@
 Changelog
 =========
 
+0.2.2
+-----
+
+- Added missing import warnings to Aito client.
+- Updated AitoClient API documentation.
+
 0.2.1
 -----
 
-- upload_entries now accepts genetors as well as lists
-- upload_entries_by_batches has been deprecated
+- AitoClient upload_entries now accepts genetors as well as lists. Example usage:
+
+  .. code-block:: python
+
+    import os
+    from aito.sdk.aito_client import AitoClient
```

Comment:
```suggestion
```

Reply:
Why should this be deleted? The idea was that the whole block is copy pastable and runnable.